src/NHSD.BuyingCatalogue.Solutions.Application/Domain/Pricing/TimeUnit.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using NHSD.BuyingCatalogue.Infrastructure;
+
+namespace NHSD.BuyingCatalogue.Solutions.Application.Domain.Pricing
+{
+    public sealed class TimeUnit : Enumerator
+    {
+        public static readonly TimeUnit PerMonth = new TimeUnit(1, "month", "per month");
+        public static readonly TimeUnit PerYear = new TimeUnit(2, "year", "per year");
+
+        public string Description { get; }
+
+        private TimeUnit(int id, string name, string description) : base(id, name)
+        {
+            Description = description ?? throw new ArgumentNullException(nameof(description));
+        }
+    }
```

Comment:
As per previous comments on these `Enumerator` types, although this one adds the description attribute I'm still not sure about it (particularly as the unit of time comes from the DB)...